### PR TITLE
CDAP-1967 Excluding org.codehaus.jackson from getting packaged in CDAP artifacts

### PR DESCRIPTION
**Pull request by poornachandra**
JIRA - https://issues.cask.co/browse/CDAP-1967

### CODE REVIEW

**Comment by bdmogal** [comment]
LGTM

**Comment by poornachandra** [comment]
Creating CDAP rpms and debs, and verified that org.codehaus.jackson artifacts are not getting packaged.

File: cdap-ui/pom.xml [hunk 1]
```diff
@@ -98,7 +98,7 @@
               <excludeGroupIds>org.apache.hbase,asm,org.apache.zookeeper,org.apache.kafka,org.apache.twill,\
                 org.apache.httpcomponents,org.apache.thrift,org.apache.flume,org.apache.avro,org.ow2.asm,\
                 com.google.inject,org.apache.hadoop,commons-codec,com.google.inject.extensions,\
-                co.cask.http</excludeGroupIds>
+                co.cask.http,,org.codehaus.jackson</excludeGroupIds>
```
Comment (gsps1) [inline cdap-ui/pom.xml]: remove extra comma?

**Comment by poornachandra** [comment]
Good catch @gsps1. Please take a look now.

**Comment by gsps1** [comment]
LGTM :+1:

**Comment by poornachandra** [comment]
Verified that this works on hdp 2.2.4.2 cluster.

**Comment by poornachandra** [comment]
Bamboo build - http://builds.cask.co/browse/CDAP-RBT330-3 is green.

**Pull request merged**